ui/packages/consul-peerings/app/components/consul/peer/components.scss

```diff
@@ -19,58 +19,54 @@
 
 %pill-pending::before {
   --icon-name: icon-running;
-  --icon-color: rgb(var(--tone-gray-800));
+  --icon-color: var(--token-color-palette-neutral-600);
 }
 %pill-pending {
-  background-color: rgb(var(--tone-strawberry-050));
-  color: rgb(var(--tone-strawberry-500));
+  background-color: var(--token-color-consul-surface);
+  color: var(--token-color-consul-foreground);
 }
 
 %pill-establishing::before {
   --icon-name: icon-running;
-  --icon-color: rgb(var(--tone-gray-800));
+  --icon-color: var(--token-color-palette-neutral-600);
 }
 %pill-establishing {
-  background-color: rgb(var(--tone-blue-050));
-  color: rgb(var(--tone-blue-500));
+  background-color: var(--token-color-palette-blue-50);
+  color: var(--token-color-palette-blue-200);
 }
 
 %pill-active::before {
   --icon-name: icon-check;
-  --icon-color: rgb(var(--tone-green-800));
+  --icon-color: var(--token-color-palette-green-400);
 }
 %pill-active {
-  background-color: rgb(var(--tone-green-050));
-  color: rgb(var(--tone-green-600));
+  background-color: var(--token-color-palette-green-50);
+  color: var(--token-color-palette-green-200);
 }
 
 %pill-failing::before {
   --icon-name: icon-x;
-  --icon-color: rgb(var(--tone-red-500));
+  --icon-color: var(--token-color-palette-red-200);
 }
 %pill-failing {
-  background-color: rgb(var(--tone-red-050));
-  color: rgb(var(--tone-red-500));
+  background-color: var(--token-color-palette-red-50);
+  color: var(--token-color-palette-red-200);
 }
 
 %pill-terminated::before {
   --icon-name: icon-x-square;
-  --icon-color: rgb(var(--tone-gray-800));
+  --icon-color: var(--token-color-palette-neutral-600);
 }
 %pill-terminated {
-  background-color: rgb(var(--tone-gray-150));
-  color: rgb(var(--tone-gray-800));
+  background-color: var(token-color-palette-neutral-200);
```

Comment:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/91899352/219776683-4519de79-6013-46d8-948a-18537b59523d.png">

I believe this is the value getting overridden causing the terminated badge to not have a background color  (screenshot below)